### PR TITLE
DX-2790 Update Landing Page

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

File: site/src/components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    var slidePaddingVar = 25;
+    var navButtonSizeVar = 50;
+    const Svg = require('@site/static/img/amoeba.svg').default;
+    const [currentIndex, setCurrentIndex] = useState(itemList.length * 2);
+    const [hasTransitionClass, setHasTransitionClass] = useState(true);
+    const [navDisabled, setNavDisabled] = useState(false);
+    const [slidePadding, setSlidePadding] = useState(slidePaddingVar);
+    const [itemWidth, setItemWidth] = useState(itemWidthVar);
+    const [itemHeight, setItemHeight] = useState(itemWidth * (2 / 3));
+    const [carouselWidth, setCarouselWidth] = useState(70);
```
Comment (ajrice6713) [inline site/src/components/Carousel.js:16]: Whats the reasoning for using static values here `70` and `2/3 width` ?

File: site/src/components/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        return () => window.removeEventListener('resize', resizeCarousel);
+    }, [])
+
+    useEffect(() => {   // make carousel infinite
+        if (currentIndex > itemList.length * 2) {   // keep index near the middle of the list when moving left
+            setNavDisabled(true);   // disable nav buttons while silently moving to different card
+            setTimeout(() => {  // disable transition animation after card slides, then snap to same card prior in the list
+                setHasTransitionClass(false);
+                setCurrentIndex(currentIndex - itemList.length);
+            }, 250)
```
Comment (matthewkmartin) [inline site/src/components/Carousel.js:75]: Where is this 250 coming from?  It seems to be repeated, could we extract this as a constant with more context as to what this value is doing?

**Review by matthewkmartin — CHANGES_REQUESTED** [review]

File: site/src/components/Carousel.js (hunk 1 above)
Comment (ckoegel) [reply site/src/components/Carousel.js:16]: they're fairly arbitrary. The 2/3 is the aspect ratio for the carousel item card and the 70 is the percent width of the carousel container. I need to set an initial state here so I set it to those values. It essentially just translates to css.

File: site/src/components/Carousel.js (hunk 2 above)
Comment (ckoegel) [reply site/src/components/Carousel.js:75]: That's the timeout in ms that we wait for the carousel to wrap around. I can make that a variable that gets reused.

File: site/src/components/Carousel.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        var itemStyle = {
+            width: `${itemWidth}px`,
+            height: `${itemHeight}px`
+        }
+
+        var imageStyle = {
+            width: `${itemContentWidth}px`,
+            height: `${itemContentWidth * .51}px`,
+            backgroundImage: `url(${imageUrl})`,
+            backgroundSize: `cover`,
```
[the remark this answers is not in the document]
Comment (ckoegel) [reply site/src/components/Carousel.js:109]: These static values can be moved to a static css class, but for the image div, I didn't define a class so this style object is the only thing affecting the div. I can pull these out into a static class though.

[the remark this answers is not in the document]
Comment (ckoegel) [reply site/src/components/Carousel.js:109]: that's a lie I do have a class for these 🤡

**Review by ajrice6713 — APPROVED** [review]

**Review by matthewkmartin — APPROVED** [review]

**Pull request merged**